api/users_test.go

```diff
@@ -31,16 +31,15 @@ const usersEndPointWithSortByEmailDesc = "http://localhost:25600/v1/users?sort=e
 const usersEndPointWithSortBy2FieldsDesc = "http://localhost:25600/v1/users?sort=forename:desc,lastname:desc"
 const usersEndPointWithSortBy2KnownFieldsAndUnknown = "http://localhost:25600/v1/users?sort=forename:desc,lastname:desc,dog"
 const userEndPoint = "http://localhost:25600/v1/users/abcd1234"
-const changePasswordEndPoint = "http://localhost:25600/v1/users/self/password"
+const changePasswordEndPoint = "http://localhost:25600/v1/users/self/password" // #nosec
```

Comment:
Consider grouping the constants
```suggestion
const (
…
    changePasswordEndPoint = "http://localhost:25600/v1/users/self/password" // #nosec
…
)
```